### PR TITLE
Create quick_sort.cpp

### PR DESCRIPTION
**Pull request by dheerajkotwani**
Algorithm for QuickSort

### CODE REVIEW

File: Algorithms/Sorting/quick_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    int i;  
+    for (i = 0; i < size; i++)  
+        cout << arr[i] << " ";  
+    cout << endl;  
+}  
+  
+// Driver Code 
+int main()  
+{  
+    int arr[] = {10, 7, 8, 9, 1, 5};
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/quick_sort.cpp]: make it dynamic not static

File: Algorithms/Sorting/quick_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    for (i = 0; i < size; i++)  
+        cout << arr[i] << " ";  
+    cout << endl;  
+}  
+  
+// Driver Code 
+int main()  
+{  
+    int arr[] = {10, 7, 8, 9, 1, 5};  
+    int n = sizeof(arr) / sizeof(arr[0]);
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/quick_sort.cpp]: no single letter variables

File: Algorithms/Sorting/quick_sort.cpp [hunk 3]
```diff
@@ -0,0 +1,61 @@
+#include <bits/stdc++.h> 
+using namespace std;  
+  
+// A utility function to swap two elements  
+void swap(int* a, int* b)  
+{  
+    int t = *a;
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/quick_sort.cpp]: no single letter variable

File: Algorithms/Sorting/quick_sort.cpp [hunk 4]
```diff
@@ -0,0 +1,61 @@
+#include <bits/stdc++.h>
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/quick_sort.cpp:14]: program name, description, same input output in comments

File: Algorithms/Sorting/quick_sort.cpp (hunk 2 above)
Comment (dheerajkotwani) [reply Algorithms/Sorting/quick_sort.cpp]: changed it

**Pull request merged**